{
  "id": "CVE-2020-14314",
  "definition_id": "oval:com.ubuntu.trusty:def:2020143140000000",
  "title": "CVE-2020-14314 on Ubuntu 14.04 LTS (trusty) - medium.",
  "description": "A memory out-of-bounds read flaw was found in the Linux kernel before 5.9-rc2 with the ext3/ext4 file system, in the way it accesses a directory with broken indexing. This flaw allows a local user to crash the system if the directory exists. The highest threat from this vulnerability is to system availability. Jay Shin discovered that the ext4 file system implementation in the Linux kernel did not properly handle directory access with broken indexing, leading to an out-of-bounds read vulnerability. A local attacker could use this to cause a denial of service (system crash).",
  "note": "",
  "severity": "Medium",
  "affected": {
    "family": "unix",
    "platform": "Ubuntu 14.04 LTS"
  },
  "packages": [
    {
      "name": "linux",
      "status": "ignored",
      "note": "out of standard support"
    },
    {
      "name": "linux-aws",
      "status": "ignored",
      "note": "out of standard support"
    },
    {
      "name": "linux-azure",
      "status": "ignored",
      "note": "out of standard support"
    },
    {
      "name": "linux-lts-xenial",
      "status": "ignored",
      "note": "out of standard support"
    },
    {
      "name": "linux-meta",
      "status": "ignored",
      "note": "out of standard support"
    },
    {
      "name": "linux-meta-aws",
      "status": "ignored",
      "note": "out of standard support"
    },
    {
      "name": "linux-meta-azure",
      "status": "ignored",
      "note": "out of standard support"
    },
    {
      "name": "linux-meta-lts-xenial",
      "status": "ignored",
      "note": "out of standard support"
    },
    {
      "name": "linux-signed",
      "status": "ignored",
      "note": "out of standard support"
    },
    {
      "name": "linux-signed-azure",
      "status": "ignored",
      "note": "out of standard support"
    },
    {
      "name": "linux-signed-lts-xenial",
      "status": "ignored",
      "note": "out of standard support"
    }
  ],
  "references": [
    {
      "id": "CVE-2020-14314",
      "source": "CVE",
      "url": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-14314"
    },
    {
      "id": "",
      "source": "REF",
      "url": "https://ubuntu.com/security/CVE-2020-14314"
    },
    {
      "id": "",
      "source": "REF",
      "url": "https://bugzilla.redhat.com/show_bug.cgi?id=1853922"
    },
    {
      "id": "",
      "source": "REF",
      "url": "https://lore.kernel.org/linux-ext4/f53e246b-647c-64bb-16ec-135383c70ad7@redhat.com/T/#u"
    },
    {
      "id": "",
      "source": "REF",
      "url": "https://ubuntu.com/security/notices/USN-4579-1"
    },
    {
      "id": "",
      "source": "REF",
      "url": "https://ubuntu.com/security/notices/USN-4578-1"
    },
    {
      "id": "",
      "source": "REF",
      "url": "https://ubuntu.com/security/notices/USN-4576-1"
    },
    {
      "id": "",
      "source": "REF",
      "url": "https://ubuntu.com/security/notices/USN-4752-1"
    }
  ],
  "public_date": "2020-09-15T20:15:00Z",
  "public_date_at_usn": "2020-09-15T20:15:00Z",
  "discovered_by": "Jay Shin",
  "rights": "Copyright (C) 2020 Canonical Ltd."
}
